browsers/duckduckgo.safariextension/js/content-script.js

```diff
@@ -51,14 +51,5 @@ function getLocation () {
 }
 
 window.onbeforeunload = ((e) => unload(e))
-window.onfocus = ((e) => sendLoadEvent(e))
```

Comment:
what was the onfocus handler for? That may be broken now if you needed that code to run on focus.